composer.json

```diff
@@ -13,7 +13,7 @@
     "require": {
         "php": ">=5.3.7",
         "christian-riesen/base32": "~1.0",
-        "simplesoftwareio/simple-qrcode" : "1.3.*"
+        "bacon/bacon-qr-code": "1.0.*"
```

Comment:
Doesn't that conflict with another PR where this dependency (original one) is moved to `suggest` part of `composer.json`?